tests/utils/config/config.go

```diff
@@ -82,6 +84,60 @@ func NewConfig() (*Config, error) {
 	return &conf, nil
 }
 
+func (c *Config) DebugTnf() (bool, error) {
+	if c.General.DebugTnf == "true" {
+		os.Setenv("LOG_LEVEL", "trace")
+
+		return true, nil
+	}
+
+	return false, nil
+}
+
+// CreateFolder verifies if a given folder exists, and creates it if it doesn't.
+func CreateFolder(folderPath string, perm fs.FileMode) (bool, error) {
+	_, err := os.Stat(folderPath)
+	if !os.IsNotExist(err) {
+		return false, err
+	}
+
+	err = os.MkdirAll(folderPath, perm)
+	if err != nil {
+		return false, err
+	}
+
+	return true, nil
+}
+
+func (c *Config) DefineLogFile(testSuite string, tcName string) *os.File {
+	folderPath := filepath.Join(c.General.ReportDirAbsPath, "Debug", testSuite)
+
+	_, err := CreateFolder(folderPath, 0755)
+
+	if err != nil {
+		panic(err)
+	}
+
+	tcFile := filepath.Join(folderPath, tcName+".log")
+
+	// if the log file already exists, remove it and create a new one.
+	if _, err := os.Stat(tcFile); err == nil {
+		err = os.Remove(tcFile)
+
```

Comment:
Recommendation: do not add extra blank lines before the "if err != nil".

Reply:
Fixed. will upload in the commit. thanks